spec/models/participant_spec.rb

```diff
@@ -20,27 +20,126 @@
       end
     end
   end
-  describe 'total score' do
-    it 'should be zero when initialized' do
+  describe 'total_score' do
+    it 'initializes to zero' do
       expect(participant.total_score).to eql 0
     end
-    it 'should increase by quiz score' do
+    it 'increases by quiz score' do
       quiz_score = create :quiz_score
       expect { participant.update quiz_score: quiz_score }
         .to change { participant.total_score }
         .by(quiz_score.score)
     end
-    it 'should increase by circle check score' do
+    it 'increases by circle check score' do
       circle_check_score = create :circle_check_score
       expect { participant.update circle_check_score: circle_check_score }
         .to change { participant.total_score }
         .by(circle_check_score.score)
     end
-    it 'should increase by onboard judging score' do
+    it 'increases by onboard judging score' do
       onboard_judging = create :onboard_judging
       expect { participant.update onboard_judging: onboard_judging }
         .to change { participant.total_score }
         .by(onboard_judging.score)
     end
   end
+  describe 'maneuver_score' do
+    it 'returns the maneuver score' do
```

Comment:
This isn't really a test of much. I agree that if you just make a participant with no objects they should have a score of 0. Better to create some maneuver_participants and make sure that there score is counted.